src/Tools/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.targets

```diff
@@ -63,6 +63,10 @@
       <_DotNetGetDocumentCommand>$(_DotNetGetDocumentCommand) $(OpenApiGenerateDocumentsOptions)</_DotNetGetDocumentCommand>
     </PropertyGroup>
 
+    <PropertyGroup>
```

Comment:
I'm a little bit nervous about pulling this out into a seperate property group since they are evaluated in order of appearance. I think it would make sense to tweak this a bit and include it in reference to `_DotNetGeDocumentCommand` like so:

```
<_DotNetGetDocumentCommand Condition=" '$(OpenApiGenerateEnvironment)' != '' " >--environment &quot;$(OpenApiGenerateEnvironment)&quot;</_DotNetGetDocumentCommand>
```